Source/Utils/ParticleUtils.H

```diff
@@ -17,6 +17,58 @@
 
 namespace ParticleUtils {
 
+    /**
+     * \brief Determine which cell a particle lies in.
+     * Returns an amrex::IntVect with the i, j, k indices of the cell the particle lies within
+     *
+     * @param[in] x the particle's x position
+     * @param[in] y the particle's y position
+     * @param[in] z the particle's z position
+     * @param[in] plo the particle lower bounds
+     * @param[in] dxi the inverse cell size
+     * @param[in] lo
+     */
+    AMREX_GPU_HOST_DEVICE AMREX_INLINE
+    amrex::IntVect getParticleCellIndex (amrex::ParticleReal xp,
+                                         amrex::ParticleReal yp,
+                                         amrex::ParticleReal zp,
+                                         amrex::GpuArray<amrex::Real,AMREX_SPACEDIM> const& plo,
+                                         amrex::GpuArray<amrex::Real,AMREX_SPACEDIM> const& dxi,
+                                         amrex::Dim3 lo) {
+        int ii = 0, jj = 0, kk = 0;
+        (void)yp, (void)zp, (void)jj, (void)kk;
```

Comment:
Instead of casting to `void` we usually do
```suggestion
        amrex::ignore_unused(yp, zp, jj, kk);
```
or you can add `[[maybe_unused]]` before the variable definition, for example, `[[maybe_unused]] int jj`.

Reply:
Yep, `[[maybe_unused]]` in the API is the modern way.
We added `amrex::ignore_unused` when we did not yet compile for C++17.

Reply:
Thanks! That's exactly what I was looking for.